src/backend/ManagementHub.Storage/Contexts/Team/DbTeamContextProvider.cs

```diff
@@ -49,4 +54,35 @@ public Task<ITeamContext> UpdateTeamAsync(NgbIdentifier ngb, TeamIdentifier team
 	{
 		return this.dbTeamContextFactory.UpdateTeamAsync(ngb, teamId, teamData);
 	}
+
+	public async Task<ITeamContext?> GetTeamAsync(TeamIdentifier teamId)
+	{
+		return await this.dbTeamContextFactory.QueryTeamsInternal(NgbConstraint.Any)
+			.Where(t => t.Id == teamId.Id)
+			.Select(DbTeamContextFactory.Selector)
+			.FirstOrDefaultAsync();
+	}
+
+	public async Task<IEnumerable<ManagerInfo>> GetTeamManagersAsync(TeamIdentifier teamId)
+	{
+		var teamDbId = teamId.Id;
+
+		var managers = await this.dbContext.TeamManagers
```

Comment:
Can you move this into dbTeamContextFactory class so that this provide doesn't depend directly on dbContext?